entities/src/main/java/com/flipkart/varadhi/entities/OrgDetails.java

```diff
@@ -0,0 +1,21 @@
+package com.flipkart.varadhi.entities;
+
+import com.flipkart.varadhi.entities.filters.OrgFilters;
+import lombok.Getter;
+
+@Getter
+
+public class OrgDetails extends MetaStoreEntity {
```

Comment:
this looks like POJO, not metastore entity. Org and OrgFilters are already stored in ZK.